client/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -1,26 +1,29 @@
 angular.module('managerApp').controller('XdslAccessCtrl', class XdslAccessCtrl {
   constructor(
-    $filter, $q, $scope, $stateParams, $templateCache, $translate, $uibModal,
-    OvhApiPackXdsl, OvhApiXdsl, OvhApiXdslIps, OvhApiXdslLines, OvhApiXdslModem,
-    OvhApiXdslNotifications, OvhApiXdslTasksCurrent, TucToast, TucToastError,
+    $filter, $q, $rootScope, $scope, $state, $stateParams, $templateCache, $translate, $uibModal,
+    OvhApiPackXdsl, OvhApiXdsl, OvhApiXdslDiagnostic, OvhApiXdslIps, OvhApiXdslLines,
+    OvhApiXdslModem, OvhApiXdslNotifications, TucToast, TucToastError, XdslTaskPoller,
     PACK, PACK_IP, REDIRECT_URLS,
   ) {
     this.$filter = $filter;
     this.$q = $q;
+    this.$rootScope = $rootScope;
     this.$scope = $scope;
+    this.$state = $state;
     this.$stateParams = $stateParams;
     this.$templateCache = $templateCache;
     this.$translate = $translate;
     this.$uibModal = $uibModal;
     this.OvhApiPackXdsl = OvhApiPackXdsl;
     this.OvhApiXdsl = OvhApiXdsl;
+    this.OvhApiXdslDiagnostic = OvhApiXdslDiagnostic.v6();
```

Comment:
It works, IMO it will be great if we inject `OvhApiXdsl` only and then use `this.OvhApiXdsl.Diagnostic`, `this.OvhApiXdsl.Ips`, and so on...
It require to update [`ovh-api-services`](https://github.com/ovh-ux/ovh-api-services/blob/master/src/xdsl/xdsl.service.js#L16-L24)